neqo-http3/src/client_events.rs

```diff
@@ -83,24 +81,19 @@ pub struct Http3ClientEvents {
     events: Rc<RefCell<VecDeque<Http3ClientEvent>>>,
 }
 
-impl RecvMessageEvents for Http3ClientEvents {
-    /// Add a new `HeaderReady` event.
-    fn header_ready(&self, stream_id: u64, headers: Vec<Header>, interim: bool, fin: bool) {
-        self.insert(Http3ClientEvent::HeaderReady {
-            stream_id,
-            headers,
-            interim,
-            fin,
-        });
-    }
-
+impl RecvStreamEvents for Http3ClientEvents {
```

Comment:
Sorry for a stupid question: is `RecvStreamEvents` only used by webtransport? what's the difference with `HttpRecvStreamEvents`?

Reply:
`RecvStreamEvents` contain functions that are more general and common for http streams and webtransport streams.
`HttpRecvStreamEvents` extends `RecvStreamEvents` and adds functions that are only specific to http streams